harold/plugins/deploy.py

```diff
@@ -278,6 +268,38 @@ def add_repo(self, repo_name):
     def remove_repo(self, repo_name):
         yield self.db.remove_repository(self.name, repo_name)
 
+    @inlineCallbacks
+    def set_deploy_hours(self, irc, start, end, tz):
+        yield self.db.set_deploy_hours(self.name, start, end, tz)
+        self.deploy_hours_start = start
+        self.deploy_hours_end = end
+        self.tz = tz
+        self.update_topic(irc)
+
+    def current_time_status(self):
+        date = datetime.date.today()
```

Comment:
This is going to be based on server localtime rather than the salon's timezone. I think you want the `.date()` of the `datetime.datetime.now(tz=self.tz)`?